src/bootstrap/server-config.js

```diff
@@ -50,6 +51,7 @@ const configSchema = joi.object({
   }).allow(null),
   basicAuthUsers: joi.object().default({}),
   plugins: joi.array().items(joi.string().required()).default(['markdown', 'image']),
+  allowedLicenses: joi.array().items(joi.string().valid(...Object.keys(spdxLicenseList))).default(Object.keys(spdxLicenseList)),
```

Comment:
Let's add this to the `Readme`, maybe including a small example, like `'CC0-1.0', 'CC-BY-4.0']`?

Reply:
Done! Thanks for reminding me each time :blush:

Reply:
😄 Sure thing!